.github/workflows/close-conflicted-prs.yml

```
@@ -0,0 +1,42 @@
+name: Close Old Conflicted PRs
+
+on:
+  schedule:
+    - cron: "0 * * * *" 
+
+jobs:
+  close_conflicted_prs:
+    runs-on: ubuntu-latest
+    steps:
+      - name: Close PRs with merge conflicts older than 3 days
+        uses: actions/github-script@v6
```

actionlint (1.7.7)

the runner of "actions/github-script@v6" action is too old to run on GitHub Actions. update the action's version to fix this issue

(action)